test/comment-regex.js

```diff
@@ -28,15 +40,21 @@ test('comment regex old spec - @', function (t) {
     '\t/*@ ', // multi line style with leading tab
     '/*@ ',   // multi line style with leading text
   ].forEach(function (x) { 
-    t.ok(comment(x, ''), 'matches ' + x) 
-    t.ok(commentWithCharSet(x, ''), 'matches ' + x + ' with charset') 
-    t.ok(commentWithCharSet(x, '', '='), 'matches ' + x + ' with charset')
+    t.ok(comment(x, '', convert.commentRegex), 'matches ' + x)
```

Comment:
ditto here .. existing tests should be touched as little as possible to ensure nothing breaks